utils/zerovec/src/error.rs

```diff
@@ -33,6 +46,27 @@ impl fmt::Display for ZeroVecError {
         }
     }
 }
+impl JsonSchema for ZeroVecError {
```

Comment:
Issue: I don't think you should need this impl. It should go away when you fix the ZeroSlice impl.